src/test-container.ts

```diff
@@ -30,7 +29,6 @@ export interface TestContainer {
   withDefaultLogDriver(): this;
   withPrivilegedMode(): this;
   withPullPolicy(pullPolicy: PullPolicy): this;
-  withAuthentication(authConfig: AuthConfig): this;
```

Comment:
Removing `withAuthentication` is a breaking change, worth calling out in changelog / release notes